### ISSUE TITLE
[Wallet Redesign] Layout

### ISSUE THREAD

**Issue by gibbsfromncis** [bitshares/bitshares-ui#2667]

**Comment by startailcoon** [comment]
My initial reflections on the layout design.

## Overall
The layout needs to accommodate whatever links we have today from the top-nav menu, like Voting, Membership stats and vesting balance. While we should try to go for a "simpler" design, we shouldn't bury features to deep down in sub pages. I'm sure we won't, but just want to have that said.

## Top menu
As you've mentioned that you will submit a mockup for mobile devices later I'll not go into detail that the top menu will have to be designed somehow to fit on a smaller display. However, I think that the top should contain account values in preferred asset.

## Side bar

Having a navigation bar is both good and bad. We moved away from it last year in the client we have today, and are today using the drop down menu on the top right. 

- Pros: Makes it much more visual and easy to see.
- Cons: It needs a replacement for mobile devices, and thus a second menu system.

If we choose to have one, we should make sure that the user can set it to minimize, like how we have the panels on the exchange today. The panel should also be possible to "View/hide on mouse over". A mobile solution could be to swipe from the side to view it. It should also be customizable to set which side you want the menu to be on.

 ## Bottom menu
The bottom menu we have today isn't included, so we lack quick features like Blockheight and switch node. We should try to make it visual that operations are done on a blockchain that progresses steadily. I'm sure we can do that without having a bottom menu.

We also have "FAQ" and "Report". We could have these elsewhere.

**Comment by sschiessl-bcp** [comment]
- Sidebar can be solved b a pinnable solution
- While Report was a nice idea, no one uses it
- Footer is not necessary to be visible at all times. Most users don't care about blocktime and connectivtiy. Could be a floating symbol on the bottom right that expands when clicked, and auto expands if error

**Comment by froooze** [comment]
I want to combine these things:
- Need Help + Beginners guide = HELP
- Membership + Invite & Earn = Membership
- Contacts + Accounts = more in extra topic

News & Voting should have an entry. 

Where do you put node info and node settings?

**Comment by sschiessl-bcp** [comment]
Yes, I would even say Voting or Governance needs to be a main point in menu

**Comment by froooze** [comment]
Where should we put, Spotlight, Asset creation? 
Signed messages and Whitelist also in User menu?
